classifiers/reference_quality/special_character_classifier/code_snippet_refinery.md

```diff
@@ -0,0 +1,27 @@
+```python
+import unicodedata
+from typing import Optional, List, Tuple
+
+ATTRIBUTE: str = "text" # only text attributes
+ALLOWED_RANGES: List[Tuple[int,int]] = None
```

Comment:
currently not possible with bricks integrator. But since optional probably still possible

Reply:
Changed this to `List[int]` now.